src/firefly/js/visualize/draw/FootprintFactory.js

```diff
@@ -19,17 +19,32 @@ const JWST_INST = new Enum(['FGS', 'MIRI', 'NIRCAM', 'NIS', 'NIRSPEC']);
 const HST_INST = new Enum(['NICMOS', 'WFPC2', 'ACS/WFC', 'ACS/HRC', 'ACS/SBC', 'WFC3/UVIS', 'WFC3/IR']);
 const SPITZER_INST = new Enum(['IRAC36', 'IRAC45']);
 
-//To add a new instrument, add it in SOFIA_INST.  It the new instrument has submenu, create a menu list like SOFIA_INST_FORCAST_TEXT
-//and then add it to SOFIA_INSTRUMENTS
-const SOFIA_INST = new Enum(['FORCAST', 'FIFI-LS']);
+//To add a new  Sofia instrument, add it in SOFIA_INST.  If the new instrument has submenu, create a menu list like
+// SOFIA_INST_FORCAST_TEXT and then add it to SOFIA_INSTRUMENTS
+const SOFIA_INST = new Enum(['FORCAST', 'FIFI-LS', 'FLITECAM', 'FPI+', 'HAEWC+']);
```

Comment:
I've noticed that `HAWC+` was misspelled here.

Reply:
@ejoliet  Good catch.  I was waiting for the dev ready to push my changes.  I just pushed my changes.  Thanks!